libraries/Microsoft.Bot.Connector/Authentication/MicrosoftAppCredentials.cs

```diff
@@ -141,7 +141,7 @@ protected override Lazy<AdalAuthenticator> BuildAuthenticator()
                 () =>
                 new AdalAuthenticator(
                     new ClientCredential(MicrosoftAppId, MicrosoftAppPassword),
-                    new OAuthConfiguration() { Authority = OAuthEndpoint, Scope = OAuthScope },
+                    new OAuthConfiguration() { Authority = OAuthEndpoint, ValidateAuthority = ValidateAuthority, Scope = OAuthScope },
```

Comment:
Will this not change the default from 'false' to 'true'?  is this backwards compatible?

Reply:
The default should be true. True means the authority is validated which should be the default. @willportnoy wanted the ability to turn this to false when the cloud wasn't Public or Gov.